kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -343,8 +348,9 @@ class NotificationManager internal constructor(
             }
 
             override fun getCurrentContentText(player: Player): CharSequence? {
-                return player.mediaMetadata.artist ?: player.mediaMetadata.albumArtist
-                    ?: (player.currentMediaItem?.localConfiguration?.tag as AudioItemHolder).audioItem.artist
+                return notificationMetadata?.artist
+                    ?: player.mediaMetadata.artist
+                    ?: player.mediaMetadata.albumArtist
```

Comment:
Okay so this should be good to go as is – we are using `notificationMetadata` for specifying metadata to be displayed in the notification.. which you can set to occur automatically or you can update to something custom

To be honest, I wonder now if displaying file metadata as a fallback should not be removed also.. in order to completely leave it up to the user.. but since that would be a breaking change, let's keep that decision for another time